.test/great_expectations/expectations/mobile/v1/mobile_metadata.json

```diff
@@ -103,7 +103,8 @@
   "meta": {
     "versions": {
       "test_suite_version": "1.0.0",
```

Comment:
Test suite has been updated, bump version to 1.0.1

Reply:
How has it changed? Adding GBQ to the expectation?

Reply:
We discussed on the call but to clarify on the record - there are changes to the entire suite as a whole. I picked a bad file to put this comment on!